pom.xml

```diff
@@ -22,6 +22,7 @@
     <module>components</module>
     <module>webclient</module>
     <module>assemblies</module>
+    <module>solr</module>
```

Comment:
We need to rethink the "position" of the module... The root of the repo seems too much for a temporary hack...